pom.xml

```diff
@@ -131,6 +131,7 @@
         <cs.google-http-client>1.34.2</cs.google-http-client>
         <cs.groovy.version>2.4.17</cs.groovy.version>
         <cs.gson.version>1.7.2</cs.gson.version>
+        <cs.solidfire.gson.version>2.6.2</cs.solidfire.gson.version>
```

Comment:
I am not entirely sure why there is solidfire changes in this PR?  Maybe I am missing something?

Reply:
No. It's not used. I remove it.